client/src/layout/menu.js

```diff
@@ -15,7 +23,15 @@ export function logoutClick() {
             }
             // Check if we need to logout of OIDC IDP
             if (galaxy.config.enable_oidc) {
-                return axios.get(`${galaxy.root}authnz/logout`);
```

Comment:
The logout function wasn't running at all originally because the provider wasn't being passed (the endpoint is defined to receive provider in buildapp). I checked and the redirect is being returned